### ISSUE TITLE
Set visible label and aria-label separately for mwc-button

### ISSUE THREAD

**Issue by ZufengW** [material-components/material-web#1428]
**Is your feature request related to a problem? Please describe.**
Request: For `mwc-button` I want to set the visible label and aria-label separately.

This is because I want to give them different values. The `label` property sets the same value for both, so it doesn't work here.

**Describe the solution you'd like**
Possible solution: Have a separate property that just sets `aria-label`.

**Describe alternatives you've considered**

Things I've tried:
 * Use both the `label` and the slot. (Doesn't work because the button shows both as visible text)
 * Set `aria-label` in addition to `label` on the `mwc-button`. (This results in my screen reader reading both the label and the aria-label.)

A workaround that I would consider: Use a custom wrapper component with hacks.
